jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java

```diff
@@ -603,32 +605,34 @@ public Tracking(int minCapacity, int maxCapacity, int maxBucketSize, long maxHea
         public RetainableByteBuffer acquire(int size, boolean direct)
         {
             RetainableByteBuffer buffer = super.acquire(size, direct);
-            Buffer wrapper = new Buffer(buffer, size);
+            TrackingBuffer wrapper = new TrackingBuffer(buffer, size);
             if (LOG.isDebugEnabled())
                 LOG.debug("acquired {}", wrapper);
             buffers.add(wrapper);
             return wrapper;
         }
 
-        public Set<Buffer> getLeaks()
+        public Set<TrackingBuffer> getLeaks()
         {
             return buffers;
         }
 
         public String dumpLeaks()
         {
             return getLeaks().stream()
-                .map(Buffer::dump)
+                .map(TrackingBuffer::dump)
                 .collect(Collectors.joining(System.lineSeparator()));
         }
 
-        public class Buffer extends RetainableByteBuffer.Wrapper
+        public class TrackingBuffer extends RetainableByteBuffer.Wrapper
```

Comment:
I still prefer this class just be called `Buffer`, as `Tracking` is already the outer class.
We will eventually improve the toString() and for now we know what DEBUG logging prints.

Reply:
Ok, done.